homeassistant/components/alexa/entities.py

```diff
@@ -50,7 +51,7 @@
     AlexaSpeaker,
     AlexaStepSpeaker,
     AlexaTemperatureSensor,
-    AlexaThermostatController,
+    AlexaThermostatController
```

Comment:
```suggestion
    AlexaThermostatController,
```
Black formatting requires commons after last items.